SparkPost/matchbox
# UX-418: Resolve axios vulnerability

### What Changed
- Resolves axios vulnerabilities
- Removes the design-token `serve` task, since we never used it
- Updates gatsby dependencies

### How To Test or Verify
- Ensure the site builds and runs as expected
- Ensure the workspace runs tests and storybook as expected

### PR Checklist
